defi/src/adaptors/handlers/storeAdaptorData/index.ts

```diff
@@ -19,6 +19,8 @@ import { sendDiscordAlert } from "../../utils/notify";
 
 
 // Runs a little bit past each hour, but calls function with timestamp on the hour to allow blocks to sync for high throughput chains. Does not work for api based with 24/hours
+const timestampAtStartofHour = getTimestampAtStartOfHour(Math.trunc((Date.now()) / 1000))
```

Comment:
i assume this is only called from cli? otherwise this will get outdated

Reply:
mainly used in jenkin runs, adapter runs max for an hour, so even if off by an hour, it is not an issue